src/box/alter.cc

```diff
@@ -44,7 +44,7 @@
 #include "tuple.h"
 #include "fiber.h" /* for gc_pool */
 #include "scoped_guard.h"
-#include "third_party/base64.h"
+#include "base64.h"
```

Comment:
It looks like you wanted to update all `#include ""` for `#include <>` across the patch but some are slipped. Or is there more complex logic behind?

Reply:
Fixed some places some places need `<>`, in other places, I fixed it by analogy in the entire patch